Services/Certificate/classes/Placeholder/Values/class.ilUserDefinedFieldsPlaceholderValues.php

```diff
@@ -1,42 +1,25 @@
-<?php
+<?php declare(strict_types=1);
 /* Copyright (c) 1998-2019 ILIAS open source, Extended GPL, see docs/LICENSE */
 
 /**
  * @author  Niels Theen <ntheen@databay.de>
  */
 class ilUserDefinedFieldsPlaceholderValues implements ilCertificatePlaceholderValues
 {
-    /**
-     * @var array
-     */
-    private $placeholder;
-
-    /**
-     * @var ilCertificateObjectHelper|null
-     */
-    private $objectHelper;
-
+    private array $placeholder;
+    private ?ilCertificateObjectHelper $objectHelper;
```

Comment:
This is not optional. Only the passed constructor argument is optional, the value stored in `$this->objectHelper` MUST never be `null`.